projects/1.16/assets/prehistoric-delight-by-nocube/pre_delight/lang/zh_cn.json

```
@@ -0,0 +1,40 @@
+{
+  "item.pre_delight.amber_sashimi": "琥珀盖饭",
+  "item.pre_delight.ammonite_escargot": "奶油烤菊石",
+  "item.pre_delight.ancient_sea_chowder": "古海浓汤",
+  "item.pre_delight.ankylo_dish": "甲龙套餐",
+  "item.pre_delight.baked_tuber": "烤块茎",
+  "item.pre_delight.boiled_tubers": "煮块茎",
+  "item.pre_delight.ceratopsian_horn_knife": "角龙犄角刀",
+  "item.pre_delight.cladophlebis_bale": "枝脉蕨捆",
+  "item.pre_delight.cooked_mooreodontus": "熟摩尔齿鲨",
+  "item.pre_delight.cooked_myledaphus": "熟磨石鳐",
+  "item.pre_delight.cooked_oyster": "熟牡蛎",
+  "item.pre_delight.cooked_prehistoric_fillet": "熟史前鱼排",
+  "item.pre_delight.crispy_herbs": "脆植拌",
+  "item.pre_delight.crunchy_zamites_pie": "脆皮查米亚派",
```

```suggestion
  "item.pre_delight.crunchy_zamites_pie": "脆皮似查米亚派",
```
`似查米亚`为一体的称呼，请注意全文修改